src/Parser/GuardianDruid/Modules/Items/Skysecs.js

```diff
@@ -0,0 +1,118 @@
+      if (this.currentCycleFRTicksLeft === 0) {
+        this.overallEffectiveHeal += this.currentCycleEffectiveHeal;
+        this.currentCycleEffectiveHeal = 0;
+        this.potentialHeal += this.lastSkysecsTick * this.currentCycleSHTicksLeft;
+      }
+      this.currentCycleFRTicksLeft = 6;
+      this.currentCycleSHTicksLeft = 3;
+    }
+  }
+
+  on_byPlayer_heal(event) {
+    const overheal = event.overheal || 0;
+    if (event.ability.guid === SPELLS.SKYSECS_HOLD_HEAL.id) {
+      this.currentCycleEffectiveHeal += event.amount;
+      this.potentialHeal += event.amount + overheal;
+      this.lastSkysecsTick = event.amount + overheal;
+      this.currentCycleSHTicksLeft -= 1;
+    }
+
+    if (event.ability.guid === SPELLS.FRENZIED_REGENERATION.id) {
+      this.currentCycleEffectiveHeal = Math.max(this.currentCycleEffectiveHeal - overheal, 0);
+      this.currentCycleFRTicksLeft -= 1;
+    }
+  }
+
+  item() {
+    const effectiveHeal = this.overallEffectiveHeal + this.currentCycleEffectiveHeal;
+    this.efficiency = effectiveHeal / this.potentialHeal;
+
+    const fightLengthSec = (this.owner.fight.end_time - this.owner.fight.start_time) / 1000;
+
+    // Skysec's heal
+    const averagePlayerHP = this.averagePlayerHPs.reduce((sum, hp) => sum + hp, 0) / this.totalCasts;
+    const effectiveHealPerCast = this.efficiency * 0.12 * averagePlayerHP;
+    const effectiveHPS = (effectiveHealPerCast * this.totalCasts) / fightLengthSec;
+
+    return {
+      item: ITEMS.SKYSECS_HOLD,
+      result: (
+        <span>
+          <dfn
+            data-tip={`
+              The heal from Skysec's hold competes directly with the heal from Frenzied Regeneration.  Because of this, any overhealing that Frenzied Regeneration does counts against the healing that Skysec's does (since that overhealing would not have been overhealing if Skysec's was not equipped). <br /><br />
+
+              The effeciency rating is calculated as the amount of "effective healing" (healing that did not affect Frenzied Regeneration) divided by the amount of "potential healing" (all healing and overhealing done by Skysec's, including lost healing due to double-tapping).<br /><br />
+
+              Effective healing: ${formatNumber(effectiveHeal)}<br />
+              Potential healing: ${formatNumber(this.potentialHeal)}<br />
+              Efficiency: ${formatPercentage(this.efficiency)}%<br />
+              Effective HPS: ${formatNumber(effectiveHPS)} HPS
+            `}
+          >
+            {formatPercentage(this.efficiency)}% / {formatNumber(effectiveHPS)} HPS
+          </dfn>
+        </span>
+      ),
+    };
+  }
+
+  suggestions(when) {
+    when(this.efficiency).isLessThan(0.9)
+      .addSuggestion((suggest, actual, recommended) => {
+        return suggest(<span>You should not use <ItemLink id={ITEMS.SKYSECS_HOLD.id} /> if you cannot get close to full effectiveness from its extra heal.</span>)
```

Comment:
My advice when writing suggestions is to say what went wrong or could be improved, and then make a suggestion. E.g.

`Your <ItemLink id={ITEMS.SKYSECS_HOLD.id} /> efficiency was {formatPercentage(this.efficiency)}%. Consider using another legendary if you cannot get close to full effectiveness from its extra heal as this would make <ItemLink id={ITEMS.SKYSECS_HOLD.id} /> a poor quality legendary.`